Makefile

```diff
@@ -16,7 +16,7 @@ REQUIREMENTS_TIMESTAMP = $(TIMESTAMPS)/.requirements.timestamp
 DEV_REQUIREMENTS_TIMESTAMP = $(TIMESTAMPS)/.dev-requirements.timestamps
 
 # Docker variables
-DOCKER_IMG_LOCAL_TAG = swisstopo/$(SERVICE_NAME):local
+DOCKER_IMG_LOCAL_TAG = swisstopo/$(SERVICE_KML):local
```

Comment:
as `SERVICE_NAME` is defined to be `service-kml` in line 5, it should stay `SERVICE_NAME` here